improver_tests/acceptance/SHA256SUMS

```diff
@@ -691,32 +691,32 @@ a723b5cc5a2ba050a33dcc569ba67b93e8043d50bbd868cfcd065d000ae52b39  ./weighted_ble
+ed4e971a00d56d771d79ad5dec90d7be8aa1dafc19811df5734c0d602e5ec57b  ./wet-bulb-temperature-integral/realizations/input.nc
+df4703b413617aff90adc15d351ac6e4488d70a570c58e2a7979dcd2459256f6  ./wet-bulb-temperature-integral/realizations/kgo.nc
+64396013800622079dd08b20b07274aeb73232220baf135738f6861bdd306f7c  ./wet-bulb-temperature/basic/enukx_pressure.nc
+634d52800e5783cd021f7af31809263e3345197e9c63184234d8348b1d637cd3  ./wet-bulb-temperature/basic/enukx_relative_humidity.nc
+2575b46b1d57dd37b3da85f7546c7cb5cc7b7637bb1a03cbee0b772b9a51c4b2  ./wet-bulb-temperature/basic/enukx_temperature.nc
+c67228e12dbb2c8f2008a47158b774953fe697f18c3eb7f6e585ad7c5c4d7b98  ./wet-bulb-temperature/basic/with_id_attr/kgo.nc
+9bdde50b0eabaf0c00f6cefee6e80a54e56fc1a1f4c55579e7352e2d0bb84c89  ./wet-bulb-temperature/basic/without_id_attr/kgo.nc
+4c27dcf1f4953f2eff731fbcc87897f65ecd018cfeac87b1525aeb2821bfe480  ./wet-bulb-temperature/global/kgo.nc
+e52d6a748419b57010810912e3933145d4823fba5c221e68e7af703d8b9e07fb  ./wet-bulb-temperature/global/pressure_input.nc
+637541033693a1ee55e73020a2b48219d340ffbbad59b9dbec531fa4ada24b16  ./wet-bulb-temperature/global/relative_humidity_input.nc
+29fd0062ca428745e6e00c0684bcd8ecde7c0883575fd6bb90afd81ef908912e  ./wet-bulb-temperature/global/temperature_input.nc
+e92e79b427929246fa669ef53327572b49ea005bdf24257c7dadca65dcab5caf  ./wet-bulb-temperature/multi_level/enukx_multilevel_pressure.nc
+9cc3ef17d4b48caf58b3faa808b2842bacc54673d1db5ddd7d800f943d811957  ./wet-bulb-temperature/multi_level/enukx_multilevel_relative_humidity.nc
+261af68b0dc16ae614f729d0cc29ecdc5789d215ee353aeb7fc03767a470e86e  ./wet-bulb-temperature/multi_level/enukx_multilevel_temperature.nc
+dd9f473941f5ef358304265e4aee7cb85bba6bf0e50d784acaeef3cd820adb5c  ./wet-bulb-temperature/multi_level/kgo.nc
 ac7539e848cfc702d6af001a2e093c1a5fd66a07dac3e63ea5ab01e4cb914df3  ./wind-gust-diagnostic/basic/kgo_average_wind_gust.nc
 228da41cf470da4d84717c5e8bea3625821562c68bb22318fa41b39e19969497  ./wind-gust-diagnostic/basic/kgo_extreme_wind_gust.nc
 dae4aacba96f24f7b0b8521a516e033d9ae38462bd1b21f0f66699ed4bc57371  ./wind-gust-diagnostic/basic/wind_gust_perc.nc
 9323a8d84bda55828b7701f485091e297dcc03b29863b945c0e8974565565d37  ./wind-gust-diagnostic/basic/wind_speed_perc.nc
-5b30f987853961d018f3993eca9e95493ca17094fd5bcf8bd6f99f8a584367f9  ./wind_direction/basic/input.nc
-4edefaaa8d34c47de172d98193c4ffb783d447d8156ea272c54c75b8c589145d  ./wind_direction/basic/kgo.nc
-35fb9b4208e5c6bc02e390018c5b5f0884a20beef352aed0f48a828e52f8d46a  ./wind_direction/global/input.nc
-d09cdd95f2c5e5e4831abb86276a25ee71f5faecef364b6708eaa9681edb55ba  ./wind_direction/global/kgo.nc
+2688c22d71c5380c905ffa31bc59ee01eaee5822a16820e39909336f29e3eea0  ./wind_direction/basic/input.nc
+687a88b013a920ca361609cdb67a6c317795c3e267021823585618b1f1943268  ./wind_direction/basic/kgo.nc
+c630ee65998bdfc7492fddf2be1d57b2c761794039abb3e904fea77559a45a13  ./wind_direction/global/input.nc
```

Comment:
This global field has issues, as shown by your plot, which I think are arising because the longitude coordinate coordinate system isn't self-descriptive, as the longitude_of_prime_meridian isn't specified when the longitude values go from 0 to 360. Before exposing this data, it might be worth considering whether we should tidy up the coordinate definition, or replace this input with an example with longitude values from -180 to 180.

Reply:
I've used new data and replaced the KGO. We will have to accept that this test is fundamentally changed, not just thinned. I've also thinned the number of realizations. I have not been doing this in general as it alters the KGO (i.e. when generating percentiles, probabilities, wind directions), but as we are replacing the KGO anyway I can shrink the files further.